tests/test_app_utils.py

```diff
@@ -171,11 +171,10 @@ def test_process_response(self):
     def test_trim_output(self):
         short_output = {'some_field': 'some_value'}
         trimmed_short = app_utils.trim_output(short_output)
-        assert (trimmed_short == json.dumps(short_output, indent=4))
+        assert (trimmed_short == {'some_field': 'some_value'})
         long_output = {'some_field': 'some_value ' * 100000}
         trimmed_long = app_utils.trim_output(long_output)
-        assert (trimmed_long != json.dumps(long_output, indent=4))
-        assert (trimmed_long.endswith('\n\n... Output truncated ...'))
+        assert trimmed_long == 'Output too large to provide on main page - see check result directly'
```

Comment:
This is where you wish you were using a variable in the other file, so you could say
```
assert trimmed_long == OUTPUT_TOO_LARGE_MESSAGE
```